docs/index.md

```diff
@@ -66,6 +59,31 @@ can be used to:
     4.  Please also use the VM argument `-ea` (enable assertions) in your Preferences
         or in your Run/Debug configurations, so that failed assertions don’t just slip through.
 
+### Updating CLDR and ICU versions
+
+> :point_right: Note: This is a mess. See <https://unicode-org.atlassian.net/browse/ICU-21757>
+
+See the top level `pom.xml` under `<properties>`.
+
+#### ICU
+
+- Every time an ICU release/prerelease (not tag) is created on GitHub, a new package is created.
+- Go to <https://github.com/orgs/unicode-org/packages?repo_name=icu> to see what versions of ICU special packages are available.
+- Update `icu.version` to the version string, such as `70.0.1-SNAPSHOT-cldr-2021-09-15`
+#### CLDR
+
+- Every time a CLDR release/prerelease (not tag) is created on GitHub, a new package is created.
+- Go to <https://github.com/orgs/unicode-org/packages?repo_name=cldr> to see what versions of CLDR packages are available.
+- Update `cldr.version` to this version string, which has 0.0.0 and a git hash in it, such as `0.0.0-SNAPSHOT-bfa39570be`
+
+#### Using custom versions of CLDR
+
+- Look at your version of CLDR's top level `pom.xml`
+- It will have a version such as `40.0-SNAPSHOT`
+- Change `cldr.version` to `40.0-SNAPSHOT` and this version will be used.
+- If you are using Eclipse, make sure CLDR and UnicodeTools are in the same workspace,
```

Comment:
question: Is it easy enough to run this at the command-line?

Ex: `cd .github/workflows`; `mvn ....`

Reply:
yes just `mvn package` - you don't need anything in `.github` to do so.  I'll add a section.